tools/build_third_party.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding=utf-8
+#
+# build_third_party.py
+# --------------------
+# This script updates the third party dependencies of deno.
+#
+# - Get Depot Tools and make sure it's in your path.
+#   http://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up
+# - You need yarn installed as well.
+#   https://yarnpkg.com/lang/en/docs/install/
+#
+# Use gclient_config.py to modify the git deps.
+# Use js/package.json to modify the npm deps.
+
+import os
+import subprocess
+import argparse
+
+root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+third_party_path = os.path.join(root_path, "third_party")
+script_name = "build_third_party.py"
```

Comment:
I think `sync_third_party.py` is a better name. What do you think?

Reply:
I agree. This script doesn't build things, but basically updates the deps.

Reply:
@kt3k actually I changed my mind- it is only syncing now, but it will be doing more "build" type things later - like creating symlinks ([see comment](https://github.com/ry/deno/pull/328#issuecomment-402462220))

Reply:
OK.

> it will be doing more "build" type things later - like creating symlinks

That's why you called this `build_*` first.